fhir-persistence-schema/src/main/java/com/ibm/fhir/schema/control/FhirResourceTableGroup.java

```diff
@@ -415,15 +417,15 @@ public Table addResourceTokenRefs(List<IDatabaseObject> group, String prefix) {
 
         // logical_resources (1) ---- (*) patient_resource_token_refs (*) ---- (0|1) common_token_values
         Table tbl = Table.builder(schemaName, tableName)
-                .setVersion(FhirSchemaVersion.V0006.vid())
+                .setVersion(FhirSchemaVersion.V0008.vid())
```

Comment:
Please try add the `schema-change` label (to either the PR or the issue or both) whenever it contains a schema change...I've found this helps when putting together the release notes.

Reply:
I've gone ahead and added it here.

Reply:
thanks